__tests__/app.test.js

```diff
@@ -40,31 +40,122 @@ describe("/api/categories", () => {
-						designer: expect.any(String),
-						comment_count: expect.any(Number),
+		});
+	});
+	describe("/api/reviews?", () => {
+		test("GET 200 - should return an object of the relevant reviews where the category is euro game", () => {
+			return request(app)
+				.get("/api/reviews?category=euro game")
+				.expect(200)
+				.then((result) => {
+					result.body.reviews.forEach((review) => {
+						expect(review).toMatchObject({
+							review_id: expect.any(Number),
+							title: expect.any(String),
+							designer: expect.any(String),
+							owner: expect.any(String),
+							review_img_url: expect.any(String),
+							category: "euro game",
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							comment_count: expect.any(Number),
+						});
 					});
 				});
-			});
+		});
+		test("GET 200 - should return an object of reviews sorted by title in descending order (default)", () => {
+			return request(app)
+				.get("/api/reviews?sort_by=title")
+				.expect(200)
+				.then((result) => {
+					expect(result.body.reviews).toBeSortedBy("title", {
+						descending: true,
+					});
+					result.body.reviews.forEach((review) => {
+						expect(review).toMatchObject({
+							review_id: expect.any(Number),
+							title: expect.any(String),
+							designer: expect.any(String),
+							owner: expect.any(String),
+							review_img_url: expect.any(String),
+							category: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							comment_count: expect.any(Number),
+						});
+					});
+				});
+		});
+		test("GET 200 - should return an object of reviews sorted by date (default) in ascending order", () => {
+			return request(app)
+				.get("/api/reviews?order=asc")
+				.expect(200)
+				.then((result) => {
+					expect(result.body.reviews).toBeSortedBy("created_at", {
+						ascending: true,
+					});
+					result.body.reviews.forEach((review) => {
+						expect(review).toMatchObject({
+							review_id: expect.any(Number),
+							title: expect.any(String),
+							designer: expect.any(String),
+							owner: expect.any(String),
+							review_img_url: expect.any(String),
+							category: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							comment_count: expect.any(Number),
+						});
+					});
+				});
+		});
+		test("ERROR 400 - should return an error if given an incorrect order", () => {
+			return request(app)
+				.get("/api/reviews?order=bob")
+				.expect(400)
+				.then((result) => {
+					expect(result.body.msg).toBe(
+						"Sorry, you inputted something incorrectly"
+					);
+				});
+		});
+		test("ERROR 400 - should return error if given a sort_by category that doesn't exist", () => {
+			return request(app)
+				.get("/api/reviews?sort_by=fred")
+				.expect(400)
+				.then((result) => {
+					expect(result.body.msg).toBe(
+						"Sorry, you inputted something incorrectly"
+					);
+				});
+		});
```

Comment:
Great work so far, but needs two more test cases 

-Status 200, for a valid category, but one with no reviews      e.g. ?category=children's games. 

-Status 404, non-existent category query,          e.g. ?category=bananas.